data-sources/atlassian-jira/config.yml

```diff
@@ -4,6 +4,12 @@ description: |
   Jira is a proprietary issue tracking product developed by Atlassian that allows bug tracking and agile project management. You can Integrate New Relic with Atlassian Jira(Cloud) and automatically create and update Jira issues.
 icon: logo.png
 install:
+  flagged:
+    type: nerdlet
+    nerdlet_id: marketplace.install-data-source
+    nerdlet_state:
+      dataSourceId: atlassian-jira
+    requires_account: false
   primary:
     link:
       url: https://docs.newrelic.com/docs/alerts-applied-intelligence/notifications/notification-integrations/#jira
```

Comment:
```suggestion
  primary:
    nerdlet:
      nerdletId: marketplace.install-data-source
      nerdletState:
        dataSourceId: atlassian-jira
      requiresAccount: false
```

Reply:
So instead of using `mode`, the type of mode is another key under primary like: 

```
primary: 
  nerdlet: 
```

And then instead of using snake casing (like `requires_account`) we use camel case in this repo (so `requiresAccount`). The same changes need to be made for `nerdlet_id` -> `nerdletId` and `nerdlet_state` -> `nerdletState`

Reply:
hi @caylahamann  updated as per suggestions , Thank you

Reply:
Hi @caylahamann ,
As per the below documentation, which one should be good to go ?
documentation: https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-sources-1

suggestions:
`
      nerdletState:
        dataSourceId: atlassian-jira
`
OR
`
      nerdletState:
        quickstartId: alpha-numeric-id-of-quickstart
`

Reply:
It is `dataSourceId` - the `dataSourceId` is needed for nerdletState, `quickstartId` has no bearing on the install framework.